### PR TITLE
Quiz App with Timer

### LINKED ISSUE #126: Quiz App with Timer
A Quiz App with Timer using HTML,CSS,JS.

This is a quiz app with timer to create a more challenging environment.

ezgif-4-d6fe98f8b3

ezgif-4-c06dbb3269

ezgif-4-4f551ac1c5

I would really appreciate if u could allow me to do the project

### PR DESCRIPTION
**Pull request by Vivek-GuptaXCode**
# Title and Issue number 

Title : Quiz App with Timer

Issue No. : #126 

Code Stack : 

Close #126 

# Checklist:

- [X] I have mentioned the issue number in my Pull Request.
- [X] I have commented my code, particularly in hard-to-understand areas
- [] I have created a helpful and easy to understand `README.md`
- [X] I have updated the Index.html file for my contribution

### CODE REVIEW

File: native/machine-coding/Quiz-app-with-timer/index.css [hunk 1]
```diff
@@ -0,0 +1,109 @@
+@import url(../../helpers/core.css);
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+  font-family: "Poppins", sans-serif;
+}
```
Comment (sadanandpai) [inline native/machine-coding/Quiz-app-with-timer/index.css:7]: no need to add the * ruleset as it is covered in core.css

File: native/machine-coding/Quiz-app-with-timer/index.js [hunk 2]
```diff
@@ -0,0 +1,6177 @@
...
+let startScreen = document.querySelector(".start-screen");
+let startButton = document.getElementById("start-button");
+let questionCount;
+let scoreCount = 0;
+let count = 11;
+let countdown;
+
+//Questions and Options array
+
+const quizArray = [
```
Comment (sadanandpai) [inline native/machine-coding/Quiz-app-with-timer/index.js:19]: please move the quiz array to a different file and import

File: native/machine-coding/Quiz-app-with-timer/index.js [hunk 3]
```diff
@@ -0,0 +1,6177 @@
...
+let displayContainer = document.getElementById("display-container");
+let scoreContainer = document.querySelector(".score-container");
+let restart = document.getElementById("restart");
+let userScore = document.getElementById("user-score");
+let startScreen = document.querySelector(".start-screen");
+let startButton = document.getElementById("start-button");
+let questionCount;
+let scoreCount = 0;
+let count = 11;
+let countdown;
```
Comment (sadanandpai) [inline native/machine-coding/Quiz-app-with-timer/index.js:15]: use const wherever no reassignment happens

**Review by sadanandpai — CHANGES_REQUESTED** [review]

File: native/machine-coding/Quiz-app-with-timer/index.css (hunk 1 above)
Comment (Vivek-GuptaXCode) [reply native/machine-coding/Quiz-app-with-timer/index.css:7]: Okay i will change it , should i create a new pull request? I am new to github

File: native/machine-coding/Quiz-app-with-timer/index.js (hunk 2 above)
Comment (Vivek-GuptaXCode) [reply native/machine-coding/Quiz-app-with-timer/index.js:19]: Ok

**Comment by Vivek-GuptaXCode** [comment]
please review the changes and suggest if any.have a great day!!

File: native/machine-coding/Quiz-app-with-timer/index.css [hunk 4]
```diff
@@ -0,0 +1,103 @@
+@import url(../../helpers/core.css);
```
Comment (sadanandpai) [inline native/machine-coding/Quiz-app-with-timer/index.css:1]: please remove this as per the new standards introduced

File: native/machine-coding/Quiz-app-with-timer/index.html [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>MCQ Quiz</title>
+    <!-- Google Font -->
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;600&display=swap"
+      rel="stylesheet"
+    />
+    <!-- Stylesheet -->
+    <link rel="stylesheet" href="index.css" />
+    <script src="../../helpers/navbar.js" type="module"></script>
```
Comment (sadanandpai) [inline native/machine-coding/Quiz-app-with-timer/index.html:12]: dont add new fonts.

import the header as done in [counter](https://github.com/sadanandpai/frontend-mini-challenges/blob/main/native/machine-coding/counter/index.html)

Also, remove the other tags

**Review by sadanandpai — CHANGES_REQUESTED** [review]
Minor changes. Please addres for merge

File: native/machine-coding/Quiz-app-with-timer/index.css (hunk 4 above)
Comment (Vivek-GuptaXCode) [reply native/machine-coding/Quiz-app-with-timer/index.css:1]: do i need to readd * rule that was previously used?

Comment (Vivek-GuptaXCode) [reply native/machine-coding/Quiz-app-with-timer/index.css]: nevermind i understood.

**Comment by Vivek-GuptaXCode** [comment]
please look into the changes made and let me know if i missed something.Have a great day!!

**Comment by Vivek-GuptaXCode** [comment]
please confirm if everything is fine.

**Comment by Vivek-GuptaXCode** [comment]
hello maintainer, i appreciate your efforts to make this repository a perfect one can u please review the changes and merge the PR if everything is fine.

**Comment by Vivek-GuptaXCode** [comment]
If u want i can create a new PR so that all unnecessary commits and error are omitted

**Pull request closed without merging**